test/distributed_exec.jl

```diff
@@ -1277,7 +1268,7 @@ if DoFullTest
 end
 
 # Test addprocs/rmprocs from master node only
-for f in [ ()->addprocs(1; exeflags=cov_in_exeflags), ()->rmprocs(workers()) ]
+for f in [ ()->addprocs(1; exeflags=test_exeflags), ()->rmprocs(workers()) ]
```

Comment:
This is testing that `addprocs` itself (and not a wrapper) throws an appropriate error when called from a worker. `exeflags` too can be removed from this invocation. Code coverage is not affected since no workers are actually created.